### PR TITLE
Update Viper Dependencies to 'v.21.07-release'

### PR DESCRIPTION
**Pull request by ArquintL**
Updates to latest Viper release.
Currently, this PR tests against the release candidate but will be updated to the release when it's out.

### CODE REVIEW

**Comment by ArquintL** [comment]
The release 'v.21.07-release' has now happened

File: src/test/resources/regressions/features/defunc/waitgroup-simple1.gobra [hunk 1]
```diff
@@ -26,6 +26,7 @@ func spawnThreads(n int) {
 	go worker(-1, wg, PredTrue!<!>) // spawns first worker
 	// { acc(wg.WaitGroupP(), 1/2) && acc(wg.WaitGroupStarted(), 1/2) && !wg.WaitMode() && wg.Token(PredTrue!<!>) }
 
+	invariant 0 <= i && i <= n
```
Comment (jcp19) [inline src/test/resources/regressions/features/defunc/waitgroup-simple1.gobra:29]: Is this invariant now required?

**Review by jcp19 — APPROVED** [review]
Looks good to me

File: src/test/resources/regressions/features/defunc/waitgroup-simple1.gobra (hunk 1 above)
Comment (ArquintL) [reply src/test/resources/regressions/features/defunc/waitgroup-simple1.gobra:29]: yes, on line 33 `i` occurs as a permission amount. This invariant is necessary such that `i` is not negative

**Pull request merged**